$NetBSD: patch-aj,v 1.2 2025/10/20 18:32:10 vins Exp $

* Bulid functions using struct utmp only when the struct utmp is
  available

--- src/logging.intpro.orig	Wed Mar 26 00:12:28 2003
+++ src/logging.intpro
@@ -1,3 +1,5 @@
+#ifdef HAVE_STRUCT_UTMP
 int              rxvt_write_bsd_utmp              __PROTO((int utmp_pos, struct utmp *wu));
 void             rxvt_update_wtmp                 __PROTO((const char *fname, const struct utmp *putmp));
+#endif
 void             rxvt_update_lastlog              __PROTO((const char *fname, const char *pty, const char *host));
